### PR TITLE
Tia's first post

### PR DESCRIPTION
**Pull request by TCarmon**
Submitting my post for our Github basics class assignment.

### CODE REVIEW

File: _posts/TCarmon/2018-05-17-TCarmon-first-post.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: post
+author: TCarmon
+title: "TCarmon's first post!"
+---
+I am a Biomedical and Health Informatics Master's student. This is the first class I am taking as part of the program. I am excited to be learning Python in this class. Currently I work full time as a Project Manager at a Clinical Research Organization.
```
Comment (eah13) [inline _posts/TCarmon/2018-05-17-TCarmon-first-post.md]: Welcome to the class! ✈️

Comment (TCarmon) [inline _posts/TCarmon/2018-05-17-TCarmon-first-post.md]: Happy I was able to edit this `code`

File: _posts/TCarmon/2018-05-17-TCarmon-first-post.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+layout: post
+author: TCarmon
+title: "TCarmon's first post!"
```
Comment (TCarmon) [inline _posts/TCarmon/2018-05-17-TCarmon-first-post.md]: Did this `work`?

**Comment by TCarmon** [comment]
Here's the link to my Turtle assignment: https://trinket.io/python/880baed927

**Pull request merged**